python/lang/security/audit/dangerous-asyncio-create-exec-tainted-env-args.yaml

```diff
@@ -0,0 +1,98 @@
+rules:
+  - id: dangerous-asyncio-create-exec-tainted-env-args
+    mode: taint
+    options:
+      symbolic_propagation: true
+    pattern-sources:
+      - patterns:
+          - pattern-either:
+              - patterns:
+                  - pattern-either:
+                      - pattern: os.environ['$ANYTHING']
+                      - pattern: os.environ.get('$FOO', ...)
+                      - pattern: os.environb['$ANYTHING']
+                      - pattern: os.environb.get('$FOO', ...)
+                      - pattern: os.getenv('$ANYTHING', ...)
+                      - pattern: os.getenvb('$ANYTHING', ...)
+              - patterns:
+                  - pattern-either:
+                      - patterns:
+                          - pattern-either:
+                              - pattern: sys.argv[...]
+                              - pattern: sys.orig_argv[...]
+                      - patterns:
+                          - pattern-inside: |
+                              $PARSER = argparse.ArgumentParser(...)
+                              ...
+                          - pattern-inside: |
+                              $ARGS = $PARSER.parse_args()
+                          - pattern: <... $ARGS ...>
```

Comment:
should we use focs-metavariable here?

Reply:
I don't think in this case focus-metavariable would apply?